content/de/blog/hacktoberfest-2021/index.md

```diff
@@ -0,0 +1,125 @@
+<form action="https://backend.datenanfragen.de/hacktoberfest" method="POST">
+Ich möchte an der Hacktoberfest 2021-Aktion des Datenanfragen.de e.&nbsp;V. über GitHub teilnehmen.
+<div class="clearfix" style="margin-bottom: 5px;"></div>
+<!-- Pattern adapted after: https://github.com/shinnn/github-username-regex/blob/0794566cc10e8c5a0e562823f8f8e99fa044e5f4/index.js#L1 -->
+<label><div class="col40"><strong>GitHub-Nutzername</strong></div><div class="col60"><input type="text" pattern="^@?[a-zA-Z\d](?:[a-zA-Z\d]|-(?=[a-zA-Z\d])){0,38}$" name="github_user" class="form-element" required></label></div>
+<div class="clearfix" style="margin-bottom: 5px;"></div>
+<label><div class="col40"><strong>E-Mail-Adresse</strong></div><div class="col60"><input type="email" name="email" class="form-element" required></label></div>
+<div class="clearfix"></div>
+<div class="form-group"><input type="checkbox" id="accept_terms" name="accept_terms" class="form-element" required><label for="accept_terms"><div style="float: left; width: 90%;">Die <a href="https://static.dacdn.de/docs/bedingungen-hacktoberfest-2021.pdf">Teilnahme- und Datenschutzbedingungen</a> habe ich zur Kenntnis genommen. Ich bin damit einverstanden.<sup class="color-teal-700">*</sup></div></label></div>
+<div class="form-group"><input type="checkbox" id="accept_us_transfers" name="accept_us_transfers" class="form-element" required><label for="accept_us_transfers"><div style="float: left; width: 90%;">Ich möchte für meine Teilnahme die US-amerikanischen Plattform GitHub (<a href="https://docs.github.com/de/github/site-policy/github-privacy-statement">Datenschutzerklärung</a>) nutzen, für die der Datenanfragen.de e.&nbsp;V. nicht das gleiche Datenschutzniveau wie in der EU garantieren kann. Es besteht das Risiko, dass meine Daten dem Zugriff durch US-Behörden unterliegen, wogegen mir möglicherweise keine wirksamen Rechtsbehelfe zur Verfügung stehen. Damit bin ich einverstanden.<sup class="color-teal-700">*</sup></div></label></div>
+<input type="hidden" name="language" value="de">
+<input type="hidden" name="year" value="2021">
+<div style="float: right; margin-top: 10px;"><input class="button button-primary" type="submit" value="Registrieren"></label></div>
+<div class="clearfix"></div>
+</form>
+</div>
+
+Du hast aber auch die Möglichkeit, ohne Nutzung von GitHub teilzunehmen. Schick uns dazu bitte eine E-Mail an <hacktoberfest@datenanfragen.de>. In der E-Mail muss der Wunsch, an der Aktion teilzunehmen, das Einverständnis mit unseren [Teilnahme- und Datenschutzbedingungen](https://static.dacdn.de/docs/bedingungen-hacktoberfest-2021.pdf) und mindestens ein Git-Patch enthalten sein.
+
+Übrigens: Wenn Du _zusätzlich_ beim offiziellen Hacktoberfest teilnehmen möchtest, musst Du Dich dafür [separat registrieren](https://hacktoberfest.digitalocean.com/). Das ist aber **nicht** nötig, um nur an unserer Aktion teilzunehmen.
+
+
+<a id="repos"></a>
+
+## Welche Repositories zählen für die Aktion?
```

Comment:
Let's also switch around the repos and recommended contributions sections (also in the English version), that's seems more sensible to me.